Sources/LoopAlgorithm/LoopQuantity.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  File.swift
+//  LoopAlgorithm
+//
+//  Created by Cameron Ingham on 11/8/24.
+//
+
+import Foundation
+
+public struct LoopQuantity: Hashable, Equatable, Comparable, Sendable {
+
+    public let unit: LoopUnit
+    public let value: Double
+    
+    public init(unit: LoopUnit, doubleValue value: Double) {
```

Comment:
Why is this external parameter name needed?

`public init(unit: LoopUnit, value: Double)`

Reply:
Another example of matching HealthKit's API here.